app/controllers/api/v2/rh_cloud/advisor_engine_config_controller.rb

```diff
@@ -7,7 +7,7 @@ class AdvisorEngineConfigController < ::Api::V2::BaseController
         api :GET, "/rh_cloud/advisor_engine_config", N_("Show if system is configured to use local iop-advisor-engine.")
         def show
           render json: {
-            use_local_advisor_engine: ForemanRhCloud.with_local_advisor_engine?,
+            use_iop_mode: ForemanRhCloud.with_insights_smartproxy?,
```

Comment:
Can we sub `iop` for `insights` ? 

@ShimShtein @vkrizan I am thinking we should change the smart-proxy feature as well. We have an inconsistency in nomenclature.

Reply:
I think my point is being missed. I am wanting to remove the use of the word Insights in favor of just using `iop`

Reply:
I don't mind either. I'm with Shim, that the feature is name Insights, like the opensource counterpart. The "on Prem" can be implied for Foreman/Satellite.